index.ts

```diff
@@ -280,6 +281,9 @@ const responseReflexEventHandler = async (): Promise<void> => {
 const talkEventHandler = (event: ResponseReflexEvent): void => {
   // Check if stream has been interrupted by the user
   const interruptCallback = (token: string, streamId: string): boolean => {
+    if (!INTERRUPTION_ENABLED) {
```

Comment:
TODO: simply pass null callback handler instead of actually doing a callback and returning false. 
Should avoid choking the event loop